deparse.c

```diff
@@ -2880,12 +2888,14 @@ sqlite_deparse_operator_name(StringInfo buf, Form_pg_operator opform)
 		}
 		else if (strcmp(cur_opname, "~~*") == 0 ||
 				 strcmp(cur_opname, "!~~*") == 0 ||
-				 strcmp(cur_opname, "~") == 0 ||
```

Comment:
`"~"` operator here can be string regex operator, not mean only bitwise operator.    
Please revert the change in this function.    

You should modify here and need to check args type to ensure that is a bitwise operator: https://github.com/pgspider/sqlite_fdw/blob/d6387a8ea33129d69c45389ee20180d809a03f03/deparse.c#L678    
(return `false` mean not push down)

Reply:
Note: you can check operation result type also by `form->oprresult` in this function.

Reply:
> Note: you can check operation result type also by `form->oprresult` in this function.

Thanks, @nxhai98 ! Implemented in https://github.com/pgspider/sqlite_fdw/pull/83/commits/d2c8e3b18c240128db35f3ccede33b93bf24e874. Please recheck.

Reply:
Thank for fixing. Please help me to resolve this thread.